src/main/java/gregtech/common/GTValuesCommon.java

```diff
@@ -0,0 +1,16 @@
+package gregtech.common;
+
+import gregtech.api.items.metaitem.MetaItem;
+
+import static gregtech.common.items.MetaItems.*;
+
+public class GTValuesCommon {
+    
+    public static final MetaItem<?>.MetaValueItem[] PUMPS = {ELECTRIC_PUMP_LV, ELECTRIC_PUMP_MV, ELECTRIC_PUMP_HV, 
+            ELECTRIC_PUMP_EV, ELECTRIC_PUMP_IV, ELECTRIC_PUMP_LUV, ELECTRIC_PUMP_ZPM, ELECTRIC_PUMP_UV};
+
+    public static final MetaItem<?>.MetaValueItem[] FLUID_REGULATORS = {FLUID_REGULATOR_LV, FLUID_REGULATOR_MV, 
+            FLUID_REGULATOR_HV, FLUID_REGULATOR_EV, FLUID_REGULATOR_IV, FLUID_REGULATOR_LUV, FLUID_REGULATOR_ZPM, 
+            FLUID_REGULATOR_UV};
+    
+}
```

Comment:
I was expecting here bit different values like pump through put, but you were able to solve these with math.

For values which are currently here it would be better to place them in `MetaItems` file as there are already some arrays of other items